src/library_browser.js

```diff
@@ -225,6 +225,35 @@ var LibraryBrowser = {
       };
       Module['preloadPlugins'].push(audioPlugin);
 
+#if WASM
+#if MAIN_MODULE
+      var wasmPlugin = {};
+      wasmPlugin['asyncWasmLoadPromise'] = new Promise(
+        function(resolve, reject) { return resolve(); });
+      wasmPlugin['canHandle'] = function(name) {
+        return !Module.noWasmDecoding && (name.endsWith('.so') || name.endsWith('.wasm'));
+      };
+      wasmPlugin['handle'] = function(byteArray, name, onload, onerror) {
+        // loadWebAssemblyModule can not load modules out-of-order, so rather
+        // than just running the promises in parallel, this makes a chain of
+        // promises to run in series.
+        this.asyncWasmLoadPromise = this.asyncWasmLoadPromise.then(
+          function() {
+            return Module.loadWebAssemblyModule(byteArray, true)
```

Comment:
`loadWebAssemblyModule` should be quoted.

However, does it need to be on `Module` at all? I think you can just call `loadWebAssemblyModule()` here.